components/style/properties/longhand/inherited_box.mako.rs

```diff
@@ -50,6 +50,131 @@ ${helpers.single_keyword("image-rendering",
                          custom_consts=image_rendering_custom_consts,
                          animatable=False)}
 
+// Image Orientation
+// https://developer.mozilla.org/en-US/docs/Web/CSS/image-orientation
+<%helpers:longhand name="image-orientation"
```

Comment:
Please add `products="none"` here, and add parse tests for it (see https://github.com/servo/servo/wiki/Property-hacking-guide#testing). Use `./mach test-unit -p style` to test.